lib/cli.ts

```diff
@@ -141,7 +171,7 @@ module.exports = function broccoliCLI(args, ui = new UI()) {
           watcher.quit();
         }
       });
-      watcher.on('buildFailure', err => {
+      watcher.on('buildFailure', (err: Error) => {
```

Comment:
is this actually error? I think we don't have the assurance, rather i believe it is `any`